astropy/utils/data.py

```diff
@@ -1017,6 +1027,12 @@ def get_pkg_data_path(*path, package=None):
         else:
             package = module.__package__
     else:
+        # Backward-compatibility for files that used to exist in astropy.utils.iers
```

Comment:
Should we (eventually) deprecate this?

Reply:
We could do it here potentially if you like, do I recommend using the variables from ``astropy_iers_data`` instead?

Reply:
I guess indeed the ideal way would be to use `IERS_B_README` etc. I think it still makes sense to get those from `astropy.utils.iers`, so that we keep `astropy_iers_data` as an implementation detail. What do you think?

Reply:
That sounds good

Reply:
Added in https://github.com/astropy/astropy/pull/14819/commits/8f469a23107d5ce1114183a0e992d08a0984aa03

Result:

```
In [2]: get_pkg_data_filename('Leap_Second.dat', package='astropy.utils.iers')
WARNING: AstropyDeprecationWarning: Accessing Leap_Second.dat in this way is deprecated, use astropy.utils.iers.IERS_LEAP_SECOND_FILE instead [astropy.utils.data]
Out[2]: '/Users/tom/Dropbox/Code/Astropy/astropy-iers-data/astropy_iers_data/data/Leap_Second.dat'
```